Symfony/CS/Tests/AbstractFixerTest.php

```diff
@@ -17,7 +17,7 @@
 class AbstractFixerTest extends \PHPUnit_Framework_TestCase
 {
     /**
-     * @expectedException        LogicException
+     * @expectedException        \LogicException
```

Comment:
I think so, for example:
https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Console/Tests/Command/CommandTest.php#L43

It helps the IDE's and to figure out where the exception is defined. There is no hard requirement for PHPUnit so there is no functional need. Let me know if you want me to revert it.